{{cookiecutter.repo_name}}/Makefile

```diff
@@ -0,0 +1,46 @@
+.DEFAULT_GOAL := validate
+
+clean:
+	find . -name '*.pyc' -delete
+	coverage erase
+	rm -rf assets coverage
+
+requirements:
+	pip install -qr requirements/local.txt --exists-action w
+
+validate_python: clean
+	DISABLE_MIGRATIONS=True coverage run --branch --source={{cookiecutter.repo_name}} ./manage.py test {{cookiecutter.repo_name}} --settings={{cookiecutter.repo_name}}.settings.test
+	coverage report
+	make quality
+
+quality:
```

Comment:
These commands should also run on manage.py and setup.py.  We should add `*.py` to these command lines.